pyroute2/ipdb/interface.py

```diff
@@ -779,9 +779,18 @@ def error(x):
             #
             # One simple way to work that around is to remove
             # secondaries first.
-            rip = sorted(removed['ipaddr'],
-                         key=lambda x: self['ipaddr'][x]['flags'],
```

Comment:
Alright, I will look from the side then. I think I've actually hit the bug in another test on python 2.

Sometimes ip addresses are not getting created. CommitException('ipaddr setup error') is raised. That happened because target addresses didn't appear after three attempts to read current ip addressses. I've added a 0.5 second sleep each time this happens and got another CommitException('target flags is not set')
```
Traceback (most recent call last):
  File "/root/pyroute2/.tox/py26/lib/python2.6/site-packages/pyroute2/ipdb/interface.py", line 888, in commit
    transaction._wait_all_targets()
  File "/root/pyroute2/.tox/py26/lib/python2.6/site-packages/pyroute2/ipdb/transactional.py", line 449, in _wait_all_targets
    raise CommitException('target %s is not set' % key)
CommitException: target flags is not set
```